client/src/main.rs

```diff
@@ -953,7 +972,11 @@ fn main() {
 
                     extract_and_execute(
                         &matches, |api, cid| {
-                            let signer = matches.signer_arg().map(get_pair_from_str).unwrap();
+                            let signer = if let Some(sig) = matches.signer_arg() {
+                                get_pair_from_str(sig)
+                            } else {
+                                panic!("please specify --signer. must be an assignee of a recent meetup")
+                            };
```

Comment:
I generally think match statements are more readable if both branches of an `if let Some(..)` statement are needed.